metasync.browser.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
```

Comment:
This shouldn't be added to the repo, but generated with Babel on `prepublish` instead.